package.json

```diff
@@ -32,35 +32,37 @@
     "boom": "^2.8.0",
     "confidence": "^1.0.0",
     "debug": "^2.2.0",
-    "dotenv": "^0.5.1",
+    "dotenv": "2.0.0",
     "glue": "^2.0.0",
-    "good": "^5.1.2",
-    "good-console": "^4.1.0",
+    "good": "7.0.1",
+    "good-console": "6.1.2",
     "handlebars": "^3.0.0",
-    "hapi": "8.6.1",
+    "hapi": "13.5.0",
     "hapi-auth-cookie": "^3.0.1",
     "highlight.js": "^8.7.0",
     "hoek": "^4.0.1",
+    "inert": "4.0.1",
     "joi": "^5.1.0",
     "jsesc": "^1.1.0",
     "lodash": "^3.3.1",
     "marked": "^0.3.4",
     "mysql": "^2.5.5",
-    "visionary": "^2.0.0",
-    "yar": "^3.0.3"
+    "vision": "4.1.0",
+    "visionary": "6.0.0",
+    "yar": "7.0.2"
   },
   "devDependencies": {
     "cheerio": "0.20.0",
     "code": "^1.4.1",
     "eslint-config-semistandard": "6.0.2",
     "eslint-config-standard": "5.3.5",
-    "eslint-plugin-promise": "1.3.2",
-    "eslint-plugin-standard": "1.3.3",
+    "eslint-plugin-promise": "2.0.0",
+    "eslint-plugin-standard": "2.0.0",
     "lab": "10.9.0",
     "nodemon": "^1.3.7",
     "prompt": "^0.2.14",
     "proxyquire": "^1.4.0",
-    "selenium-webdriver": "2.47.0",
```

Comment:
I originally locked selenium-webdriver at this version because I read Sauce Labs only supported it but then today I found this referencing 2.53 https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-SeleniumDriverVersion